src/containers/cooperation-details/cooperation-from-scratch/CooperationFromScratch.constans.tsx

```diff
@@ -9,3 +9,5 @@ export const cooperationTranslationKeys = [
 ]
 
 export const initialSort = { order: SortEnum.Desc, orderBy: 'updatedAt' }
+
+export const myCooperations = '/my-cooperations'
```

Comment:
yeah we have in authRoutes, but i presume it will be better if it be independent

Reply:
> yeah we have in authRoutes, but i presume it will be better if it be independent

please look at the our project that we use authRoutes for navigation. not own constant property.
Why do we need repeat it ?

Reply:
done